### PR TITLE
Enable auto-submitting package to IBS

### PR DESCRIPTION
**Pull request by shundhammer**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -9,18 +9,39 @@ require "packaging/configuration"
...
 
   # defined in Rakefile in https://github.com/openSUSE/packaging_rake_tasks
-  conf.obs_api = "https://api.opensuse.org/"
-  conf.obs_project = "filesystems:snapper"
-  conf.obs_target = "openSUSE_Factory"
-  conf.obs_sr_project = "filesystems:snapper"
+  if yast_submit == :sle12sp3
+    # IBS: Devel:YaST:Head/snapper
+    # see https://github.com/yast/yast-rake/blob/master/data/targets.yml
+    conf.obs_api = "https://api.suse.de/"
```
Comment (imobachgs) [inline Rakefile]: This configuration is not the same defined in https://github.com/yast/yast-rake/blob/master/data/targets.yml#L30 ?

File: Rakefile [hunk 2]
```diff
@@ -9,18 +9,39 @@ require "packaging/configuration"
...
+  else
+    # OBS: filesystems:snapper/snapper
+    conf.obs_api = "https://api.opensuse.org/"
+    conf.obs_project = "filesystems:snapper"
+    conf.obs_sr_project = "filesystems:snapper"
+    conf.obs_target = "openSUSE_Factory"
+  end
+end
+
+desc 'Show configuration'
```
Comment (imobachgs) [inline Rakefile]: Not as part of this PR, it would be nice to move this code to yast-rake. What do you think?

File: Rakefile [hunk 3]
```diff
@@ -18,19 +18,16 @@ Packaging.configuration do |conf|
...
-    conf.obs_sr_project= "SUSE:SLE-12-SP3:GA"
-    conf.obs_target = "sle12_sp3"
-  else
+  if yast_submit == :factory
     # OBS: filesystems:snapper/snapper
     conf.obs_api = "https://api.opensuse.org/"
     conf.obs_project = "filesystems:snapper"
     conf.obs_sr_project = "filesystems:snapper"
     conf.obs_target = "openSUSE_Factory"
+  else
```
Comment (imobachgs) [inline Rakefile]: You don't need the `else` branch.

File: Rakefile [hunk 4]
```diff
@@ -9,18 +9,36 @@ require "packaging/configuration"
...
+    # Using values from
+    # https://github.com/yast/yast-rake/blob/master/data/targets.yml
+    # loaded by Yast::Tasks.submit_to()
+  end
+end
+
+desc 'Show configuration'
+task :show_config do
+  Packaging.configuration do |conf|
+    print "API: #{conf.obs_api}\n"
```
Comment (imobachgs) [inline Rakefile]: If you prefer, you can use "puts" and forget about the "\n". But I don't have a strong opinion about it.

Comment (shundhammer) [inline Rakefile]: Yes, right; in a previous version, I had a headline with more complex formatting, but that one is gone now.

**Comment by imobachgs** [comment]
LGTM, thanks!

**Comment by imobachgs** [comment]
Still LGTM :) Thanks!

**Pull request merged**